integrations/sagemaker/timestreamquery.py

```diff
@@ -56,12 +56,15 @@ def parseScalar(c_type, data):
     elif (c_type == "INTEGER"):
         return int(data)
     elif (c_type == "BOOLEAN"):
-        return bool(data)
+        return toBool(data)
     elif (c_type == "TIMESTAMP"):
         return data
     else:
         return data
 
+def toBool(data):
+    return str(data).lower() in ('y', 'yes', 't', 'true', 'on', '1')
```

Comment:
Should we look into use cases where a Timestream query returns BOOLEAN scalar type and  addresses those use cases that needs to be handled?

```
aws timestream-query query --query-string "SELECT regexp_like('example', 'ex') "
{
    "Rows": [
        {
            "Data": [
                {
                    "ScalarValue": "true"
                }
            ]
        }
    ],
    "ColumnInfo": [
        {
            "Name": "_col0",
            "Type": {
                "ScalarType": "BOOLEAN"
            }
        }
...
    ]
}
```

```
aws timestream-query query --query-string "SELECT regexp_like('example', 'ex1') "
{
    "Rows": [
        {
            "Data": [
                {
                    "ScalarValue": "false"
                }
            ]
        }
    ],
    "ColumnInfo": [
        {
            "Name": "_col0",
            "Type": {
                "ScalarType": "BOOLEAN"
            }
        }
    ],
   ...
}
```

Reply:
@sethusrinivasan Thank you for your comment. The function and test case have been modified.